pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -0,0 +1,110 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Import Project="$(MSBuildProjectName).props" />
+
+  <PropertyGroup>
+    <IsLineupPackage Condition="'$(PackageTargetRuntime)' == ''">true</IsLineupPackage>
+    <PreventImplementationReference Condition="'$(PackageTargetRuntime)' != ''">true</PreventImplementationReference>     
+    <TargetFrameworkName>uap</TargetFrameworkName>
+    <TargetFrameworkVersion>10.1</TargetFrameworkVersion>
+    <TargetFramework>$(TargetFrameworkName)$(TargetFrameworkVersion)</TargetFramework>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+
+    <RefBinDir>$(UAPPackageRefPath)</RefBinDir>
+    <LibBinDir>$(UAPPackageRuntimePath)</LibBinDir>
+
+    <NETStandardLibraryPackage>NETStandard.Library2</NETStandardLibraryPackage>
```

Comment:
Can you remove these and use the versions that are coming from dependencies.props? We shouldn't beusing NETSTandard.Library2 any more.